purchase_order_approved/models/purchase_order.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    state = fields.Selection(selection_add=[('approved', 'Approved')])
+    # TODO: inhterit state but adding approved state in a position after 'to
+    # aprove' state.
```

Comment:
Yes, but at the moment I think there is no way without overriding the standard field. :confused: